main.tf

```diff
@@ -1,13 +1,28 @@
-resource "random_integer" "example" {
+# -----------------------------------------------------------------------------------------------------------------------
+# Subscribe the Acccount to Security Hub
+# -----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_account" "enable_us_east_1" {
```

Comment:
I'm definitely leaning towards the module being responsible for one region, and instead relying on SCP to limit the deployable regions. There's usually only a handful of regions a customer/account operates in, which is very manageable to define in our configuration file. If we use this pattern of explicitly defining each reach, then we need to support 24 regions (as it's a public module), presumably requiring code generation.